### ISSUE TITLE
Is the floor_loss calculated correctly?

### ISSUE THREAD

**Issue by TCQian** [JonathonLuiten/Dynamic3DGaussians#53]
```
losses['floor'] = torch.clamp(fg_pts[:, 1], min=0).mean()
```

the existing code seems to suggest penalising points that are above the floor, forcing foreground points to fall under the floor ground.

do you instead meant to negate the loss? e.g.
```
losses['floor'] = torch.clamp(-fg_pts[:, 1], min=0).mean()
```
